jina/parsers/orchestrate/runtimes/head.py

```diff
@@ -25,3 +27,12 @@ def mixin_head_parser(parser):
         type=str,
         help='dictionary JSON with a list of connections to configure',
     )
+
+    gp.add_argument(
+        '--disable-reduce',
+        action='store_true',
+        default=False,
+        help='Disable the built-in reduce mechanism for merging parallel branches in a Flow'
```

Comment:
I think this one should be shown and not hidden. 

I also would write something in the lines:

`help='Disable the built-in reduce mechanism, set this if the reduction is to be handled by the Executor connected to this Head'`